pins/drivers.py

```diff
@@ -125,6 +125,14 @@ def load_data(
             try:
                 import rdata
 
+                # Can be removed once support for Python 3.8 is dropped.
+                # The issue is that the last `rdata` available on Python 3.8 was v0.9,
+                # which doesn'thave the same `rdata.read_rds()` function.
+                # See https://github.com/rstudio/pins-python/pull/265
+                if not hasattr(rdata, "read_rds"):
+                    parsed = rdata.parser.parse_file(f)
+                    rdata.conversion.convert(parsed)
+
                 return rdata.read_rds(f)
```

Comment:
ah, apologies-- I wasn't clear enough trying to explain this; instead of checking via `hasattr`, we can replace the entire `rdata` chunk with `rdata.parser.parse_file()`+`rdata.conversion.convert()`. The `rdata.read_rds()` just calls those 2 under the hood. Something like this should work ⏬ 

```python
try:
    import rdata

    parsed = rdata.parser.parse_file(f)
    return rdata.conversion.convert(parsed)
except ModuleNotFoundError:
# ... 
```

Reply:
Happy to do that - my only concern would be if the `rdata.read_rds()` function ever changed in the future to include additional or different aspects.

I think a compromise is just to explain in a comment that the two lines are meant to be equivalent to `rdata.read_rds()` and can be swapped out once Python 3.8 support is dropped.

Reply:
The docs do consider [`read_rds` to be a convenience function](https://rdata.readthedocs.io/en/latest/modules/rdata.read_rds.html#read-rds), which leads me to think it's okay for now, but that's a completely valid concern- it doesn't mean it will stay that way. I agree the comment is a good compromise!